docs/src/main/sphinx/security.rst

```diff
@@ -27,12 +27,13 @@ Authentication
   :maxdepth: 1
 
   security/authentication-types
-  security/ldap
   security/password-file
+  security/ldap
   security/salesforce
   security/oauth2
   security/kerberos
   security/cli
+  security/jwt-auth
```

Comment:
Let's name this just `jwt` to match the others

Reply:
Renamed.